### ISSUE TITLE
wikidetails.template.yml is not present

### ISSUE THREAD

**Issue by HQ1791** [TIBHannover/ontology2smw#64]
I am having basic problem of setting ontology2smw with my wiki. I have installed ontolong2smw and it directly created wikidetails.yml with some predefine things:
`host: 192.168.100.100
path: /w/
scheme: http
username: Admin@ontology2smwbot # bot username
password: botpasswordbotpasswordbotpassword`

When i change the username and password here and try to load the given aeon ontology to my wiki. it give me this error: 
![Capture](https://github.com/andrecastro0o/ontology2smw/assets/129038344/acf71a62-cabc-4c6b-b4bc-b4632974d26b)

I do not know how to set this up so it start working with my wiki. Any help would be appreciated.

Thank you!

**Comment by andrecastro0o** [comment]
Hi @HQ1791 ,
I haven't used or worked on this repo for a while. And at the top of my head I am not sure what is the issue here.

But reading back the [README.md](README.MD) section *To write wiki pages*: 

**wikidetails.yml & wiki write access**
* Ensure user your wiki user account belongs to bot group: see wiki page `Special:UserRights`
* Create a bot password in wiki page: `Special:BotPasswords`
* <s>**copy** `wikidetails.template.yml` as `wikidetails.yml` and fill in bot name and password: </s> this is incorrect (need to change it), it should say: *edit  `wikidetails.yml`  with your wiki's details: host, path, scheme, username, password*
* **give the bot appropriate rights** in `Special:UserRights` page:  basic, editinterface, editpage, editprotected, createeditmovepage, highvolume

If you post here the URL for your wiki's Main_Page I can help you with editing the wikidetails.yml for your use-case.

Also if you can write down, the following details, it could help me to point you in the right direction:
* Mediawiki version
* SMW version
* output from `pip freeze`

Hope it helps

**Comment by HQ1791** [comment]
Hi,

Thank you for your reply! Here are the details.

>>If you post here the URL for your wiki's Main_Page I can help you with editing the wikidetails.yml for your use-case.

-- localhost/mediawiki/index.php/Main_Page

>>Also if you can write down, the following details, it could help me to point you in the right direction:

- Mediawiki version -- 1.39.4
- SMW version -- 4.1.1
- output from pip freeze
`absl-py==1.4.0
aiofiles==0.7.0
aiohttp==3.8.4
aiosignal==1.3.1
altair==4.2.2
anyio==3.6.1
argilla==1.3.2
asgiref==3.5.2
astunparse==1.6.3
async-lru==1.0.3
async-timeout==4.0.2
atomicwrites==1.4.1
attrs==22.1.0
backoff==2.2.1
beautifulsoup4==4.12.0
Brotli==1.0.9
bs4==0.0.1
cachetools==5.3.0
certifi==2022.12.7
chardet==3.0.4
charset-normalizer==2.1.0
chatGPT==0.3.17
chromadb==0.3.21
click==8.1.3
clickhouse-connect==0.5.22
colorama==0.4.5
connegp @ git+https://github.com/surroundaustralia/Connegp2.git@08cf189bbc8ed6f1c076e6351689e03916ff2a9f
contourpy==1.0.7
cycler==0.11.0
dataclasses-json==0.5.7
Deprecated==1.2.13
dill==0.3.6
docstring-parser==0.15
duckdb==0.7.1
entrypoints==0.4
et-xmlfile==1.1.0
faiss-cpu==1.7.4
fake-useragent==1.1.3
fastapi==0.92.0
fedsearch @ git+https://github.com/surroundaustralia/vocprez-fedsearch.git@c5618c385d7054fb02dc0ca4599a8321ca0fef06
ffmpy==0.3.0
filelock==3.10.6
flake8==6.0.0
Flask==2.2.3
Flask-Compress==1.13
Flask-Markdown==0.3
flask-paginate==2022.1.8
flatbuffers==23.3.3
fonttools==4.39.2
frozenlist==1.3.3
fsspec==2023.3.0
gast==0.4.0
geojson-rewind==1.0.3
google-api-core==2.11.0
google-api-python-client==2.86.0
google-auth==2.17.1
google-auth-httplib2==0.1.0
google-auth-oauthlib==1.0.0
google-pasta==0.2.0
google-search-results==2.4.2
googleapis-common-protos==1.59.0
gradio==3.23.0
greenlet==2.0.1
grpcio==1.53.0
gunicorn==20.1.0
h11==0.12.0
h5py==3.8.0
hnswlib==0.7.0
httpcore==0.13.7
httplib2==0.22.0
httptools==0.5.0
httpx==0.20.0
huggingface-hub==0.13.3
idna==2.10
iniconfig==1.1.1
isodate==0.6.1
itsdangerous==2.1.2
jax==0.4.8
Jinja2==3.0.3
joblib==1.2.0
jsonschema==4.17.3
keras==2.12.0
kiwisolver==1.4.4
langchain==0.0.155
langflow==0.0.65
libclang==16.0.0
linkify-it-py==2.0.0
llama-cpp-python==0.1.23
lxml==4.9.2
lz4==4.3.2
Markdown==3.2.1
markdown-it-py==2.2.0
MarkupSafe==2.1.1
marshmallow==3.19.0
marshmallow-enum==1.5.1
matplotlib==3.7.1
mccabe==0.7.0
mdit-py-plugins==0.3.3
mdurl==0.1.2
ml-dtypes==0.0.4
monotonic==1.6
mpmath==1.3.0
msg-parser==1.2.0
multidict==6.0.4
mwclient==0.10.1
mwparserfromhell==0.6.5
mypy-extensions==1.0.0
networkx==3.1
nltk==3.8.1
numexpr==2.8.4
numpy==1.23.5
oauthlib==3.2.2
olefile==0.46
ontology2smw==0.1.0
openai==0.27.2
openapi-schema-pydantic==1.2.4
openpyxl==3.1.2
opt-einsum==3.3.0
orjson==3.8.8
owlready2==0.44
packaging==21.3
pandas==1.5.3
Pillow==9.4.0
pipeline==0.1.0
playwright==1.32.0
pluggy==1.0.0
posthog==3.0.1
prettytable==2.5.0
protobuf==4.22.1
psycopg2-binary==2.9.6
py==1.11.0
pyarrow==11.0.0
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycodestyle==2.10.0
pydantic==1.9.1
pydub==0.25.1
pyee==9.0.4
pyflakes==3.0.1
Pygments==2.14.0
pyldapi==3.13
pypandoc==1.11
pyparsing==2.4.7
pypdf==3.8.1
pyreadline3==3.4.1
pyrsistent==0.19.3
pysrt==1.1.2
pytest==7.1.2
python-dateutil==2.8.1
python-docx==0.8.11
python-dotenv==1.0.0
python-magic==0.4.27
python-multipart==0.0.5
python-pptx==0.6.21
pytz==2023.2
PyYAML==6.0
rdflib==6.2.0
rdflib-jsonld==0.6.2
regex==2023.3.23
requests==2.29.0
requests-oauthlib==1.3.1
rfc3986==1.5.0
rich==13.3.3
rsa==4.9
scikit-learn==1.2.2
scipy==1.10.1
semantic-version==2.10.0
sentence-transformers==2.2.2
sentencepiece==0.1.99
Shapely==1.8.2
six==1.16.0
sniffio==1.2.0
soupsieve==2.4
SPARQLWrapper==1.8.5
spotlight==3.1.0
SQLAlchemy==1.4.39
SQLAlchemy-Utils==0.38.3
starlette==0.25.0
starlette-core==0.0.1
sympy==1.11.1
tenacity==8.2.2
tensorboard==2.12.1
tensorboard-data-server==0.7.0
tensorboard-plugin-wit==1.8.1
tensorflow==2.12.0
tensorflow-estimator==2.12.0
tensorflow-intel==2.12.0
tensorflow-io-gcs-filesystem==0.31.0
termcolor==2.2.0
threadpoolctl==3.1.0
tiktoken==0.4.0
tokenizers==0.13.3
tomli==2.0.1
toolz==0.12.0
torch==2.0.0
torchvision==0.15.1
tqdm==4.65.0
transformers==4.27.4
typer==0.7.0
types-PyYAML==6.0.12.9
typing==3.7.4.3
typing-inspect==0.8.0
typing_extensions==4.5.0
uc-micro-py==1.0.1
unstructured==0.5.13
uritemplate==4.1.1
urllib3==1.26.15
uvicorn==0.20.0
watchfiles==0.19.0
wcwidth==0.2.5
websockets==11.0.2
Werkzeug==2.2.3
wikipedia==1.4.0
wrapt==1.14.1
WTForms==3.0.1
XlsxWriter==3.1.0
yarl==1.8.2
youtube-transcript-api==0.6.0
zstandard==0.21.0`
